MatomoTracker/EventSerializer.swift

```diff
@@ -73,13 +78,21 @@ fileprivate extension Event {
                 URLQueryItem(name: "c_p", value: contentPiece),
                 URLQueryItem(name: "c_t", value: contentTarget),
                 URLQueryItem(name: "c_i", value: contentInteraction),
+                
+                URLQueryItem(name: "ec_id", value: orderId),
+                URLQueryItem(name: "revenue", value: orderRevenue != nil ? "\(orderRevenue!)" : nil),
+                URLQueryItem(name: "ec_st", value: orderSubTotal != nil ? "\(orderSubTotal!)" : nil),
+                URLQueryItem(name: "ec_tx", value: orderTax != nil ? "\(orderTax!)" : nil),
+                URLQueryItem(name: "ec_sh", value: orderShippingCost != nil ? "\(orderShippingCost!)" : nil),
+                URLQueryItem(name: "ec_dt", value: orderDiscount != nil ? "\(orderDiscount!)" : nil),
                 ].filter { $0.value != nil }
 
             let dimensionItems = dimensions.map { URLQueryItem(name: "dimension\($0.index)", value: $0.value) }
             let customItems = customTrackingParameters.map { return URLQueryItem(name: $0.key, value: $0.value) }
             let customVariableItems = customVariables.count > 0 ? [URLQueryItem(name: "_cvar", value: customVariableParameterValue())] : []
+            let ecommerceOrderItemsAndFlag = orderItems.count > 0 ? [URLQueryItem(name: "ec_items", value: orderItemParameterValue()), URLQueryItem(name: "idgoal", value: "0")] : []
```

Comment:
Hm, the `idgoal` value is very interesting. I am a little scared that this might (in the future) break the goal feature. I will think about this, if we can solve this any different.